### PR TITLE
added wrapper function for json parsing

### LINKED ISSUE #8496: Add wrapper around JSON.parse
WIll you make a wrapper function around `JSON.parse` and use that wherever JSON.parse is used directly so that we don't need to add this same logic everywhere?

_Originally posted by @MUzairS15 in https://github.com/meshery/meshery/pull/8494#pullrequestreview-1581055156_

### PR DESCRIPTION
**Pull request by EraKin575**
**Notes for Reviewers**

This PR fixes #8496

Added a wrapper function for json parsing

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 00eeb6b19fc07dbe9239ebf8aef56ef31ab929ac at: https://64e31b7b7de90c60886c42f1--meshery-docs-preview.netlify.app

**Review by leecalcote — CHANGES_REQUESTED** [review]
1. Don't change older versions of the docs. Those snapshots are frozen in time.
2. Don't intermix json.parse for docs (not part of the original request) with the originally requested Meshery UI changes.

**Comment by EraKin575** [comment]
@leecalcote so I have to wrap every JSON.parse except the docs, right?

**Comment by leecalcote** [comment]
> @leecalcote so I have to wrap every JSON.parse except the docs, right?

@EraKin575 you have to execute on the original request by making changes to Meshery UI.

**Comment by l5io** [comment]
🚀 Preview for commit 2e9e30bef184f91c0369085ca7490b68ab9d39aa at: https://64e49194dafe34027b6e5472--meshery-docs-preview.netlify.app

File: docs/assets/js/lightbox.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import jsonParse from "../../../ui/components/ConnectionWizard/helpers/parseJson";
```
Comment (leecalcote) [inline docs/assets/js/lightbox.js:1]: No docs updates requested.

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by vishalvivekm** [comment]
@EraKin575 Let's discuss this on Meshery Dev call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#) if you would. :)

**Comment by EraKin575** [comment]
sure thing @vishalvivekm

**Review by aabidsofi19 — CHANGES_REQUESTED** [review]
@EraKin575   rename this to `safeJsonParse` and  add jsDoc comments  . then we are good to merge

**Comment by leecalcote** [comment]
@EraKin575 merge conflict...

**Comment by l5io** [comment]
🚀 Preview for commit efd06baf552c9ea5e951adc67c8f3f8c5379edb1 at: https://64fcba2f130f941068f25ddf--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit fc508dba5b7e312a7ec1919671b5879bdbbbea24 at: https://64fd8f6bfa00dc0452fbc0af--meshery-docs-preview.netlify.app

**Comment by EraKin575** [comment]
@aabidsofi19

**Comment by l5io** [comment]
🚀 Preview for commit 9a5c41eee00c1852ec781f566ff19b7bd7560b91 at: https://6502c3245cd4a732161e7251--meshery-docs-preview.netlify.app

**Comment by sudhanshutech** [comment]
@EraKin575 there is still docs changes which is not needed, revert any change made to docs

**Comment by l5io** [comment]
🚀 Preview for commit d9b11d4a41f69e9ed9794acf0b164c41b53d4507 at: https://6506b49b0c4f2c173f04bda9--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 0ce4909521c326927b1ec38b671714555d473585 at: https://6506b57c30d1f517158a9d19--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit b13aeb390bd02e547ab857e63f1490c2ad429932 at: https://6506c49dfbb37b1e97cf0b5d--meshery-docs-preview.netlify.app

**Pull request closed without merging**